ETC.tsv

```diff
@@ -6172,7 +6172,7 @@ ETC_20150317_006172	to move to the dungeon entrance. {nl} Be careful when you're
 ETC_20150317_006173	Gems can be obtained by hunting monsters and bosses in dungeons. {nl} Be careful, as the gems may be lost when you die.
 ETC_20150317_006174	Equipment
 ETC_20150317_006175	Sub-Weapon
-ETC_20150317_006176	Consumable
+ETC_20150317_006176	Consume
```

Comment:
This change should be reverted:
https://cdn.discordapp.com/attachments/80317349386780672/85134269197848576/consumable.png

Otherwise, we will be breaking the pattern on the categories.